docs/os/security.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Security considerations
+parent: Operating System
+nav_order: 330
+---
+## Security considerations
+
+As a networked device, reachable from the outside, the attack surface of the BitBox Base needs to be minimized as much as possible. This is why we use our BitBox App as the dedicated user interface for node management. By default, only one port is exposed on the Base to communicate over an end-to-end encrypted channel with the App, providing all functionality.
```

Comment:
(Optional) Perhaps `our BitBox App` should link somewhere, e.g https://github.com/digitalbitbox/bitbox-wallet-app/?